crates/aptos/src/move_tool/mod.rs

```diff
@@ -594,6 +598,89 @@ impl CliCommand<TransactionSummary> for PublishPackage {
     }
 }
 
+/// Publishes the modules in a Move package to the Aptos blockchain under a resource account
+#[derive(Parser)]
+pub struct CreateResourceAccountAndPublishPackage {
+    #[clap(long, parse(try_from_str=crate::common::types::load_account_arg))]
+    pub(crate) account: AccountAddress,
+
+    #[clap(long)]
+    pub(crate) seed: String,
```

Comment:
This brings up an interesting challenge - currently you can't use this same command twice with the same seed right? I wonder if we should just default to the origin account's nonce if this is not provided?

Reply:
cannot use this command twice with the same seed... but I thought the second time we need to publish the package under the resource account's module, we'd just get the signer from the `source_account` and do a regular `publish_package`? 

do we have a `nonce` for the original account? or are you suggesting that we just randomly generate something as the seed if this is not provided?

Reply:
Yeah the case I'm thinking of here is deploying multiple different resource accounts from the same origin account. May be that's not as common of a use case and we can revisit later if needed

Reply:
passing in a seed seems not user friendly. We may want an automatically allocated seed similar to GUID per account